src/components/common/UserArticleCardComponent.js

```diff
@@ -1,67 +1,92 @@
 import DislikeComponent from './DislikeComponent';
 import CommentIconComponent from './CommentIconComponent';
+import Button from '../../styles/styledComponents/Button.jsx';
 import TimerComponent from './TimerComponent';
 import BookmarkComponent from './BookmarkComponent';
 
 const UserArticleCard = ({
   img,
   articleTitle,
+  articleBody,
   tags,
+  slug,
   likeCount,
   dislikeCount,
   commentCount,
   timeCount
-}) => (
-  <Fragment>
-<Card style={{ width: '70rem', height: 'auto' }} className = 'mb-3 article-card'>
-<div className = 'w-25 Article-image-wrapper pb-0'>
-< img src= {img} className='img-fluid'/>
-  </div>
-  <Card.Body className = 'w-75 pb-0'>
-    <div className = 'article-title'>
-     <b className='text-body'>{articleTitle}</b>
-    </div>
-    <div className = 'tags-wrapper mb-4'>
-   {tags.split(',')
-     .map((eachTag, index) => <BadgeComponent key={index} eachTagDetail={eachTag} />)}
-    </div>
+}) => {
+  const createMarkup = () => ({ __html: articleBody });
+  return (
+    <Fragment>
+      <Card className="user-article-card">
+        <div className="article-image-wrapper">
+          <img src={img} className="img-fluid" />
+        </div>
 
-<div className='container px-0'>
-   <div className='row no-gutters'>
-           <div className='col-9 pt-3'>
-              <div className='row'>
-                 { timeCount === 'false' ? ''
-                   : <span className='col-4'>
-                <TimerComponent timeCount={timeCount} />
-              </span>}
-                 <span className="col-2">
-                   <LikeComponent likeCount={likeCount} color='blue' />
-                 </span>
-                 <span className="col-2">
-                  <DislikeComponent dislikeCount={dislikeCount} color='black' />
-                 </span>
-                 <span className="col-2">
-                 <CommentIconComponent commentCount={commentCount}/>
-                 </span>
-              </div>
+        <Card.Body className="article-card-body">
+          <NavLink to={`/articles/${slug}`}>
+            <div className="article-title-div">
+              <b className="text-body">{articleTitle}</b>
+            </div>
+            <div className="article-body" dangerouslySetInnerHTML={createMarkup()} />
+          </NavLink>
+          {tags && (
+            <div className="tags-wrapper">
+              {tags.split(',').map((eachTag, index) => (
+                <BadgeComponent key={index} eachTagDetail={eachTag} />
+              ))}
             </div>
-          <div className="col-3">
-          { timeCount === 'false'
-            ? <BookmarkComponent color='#FCC133'/>
-            : <button type="button" className="btn btn-outline-primary">Primary</button>}
+          )}
 
+          <div className="container">
+            <div className="row no-gutters">
+              <div className="icons">
+                <div className="row">
+                  {timeCount === 'false' ? (
```

Comment:
Shortcircuiting here is more ideal fam.
```suggestion
                  {timeCount !== 'false' && (
                     <span className="icon-item timer">
                       <TimerComponent timeCount={timeCount} />
                     </span>
                  )}
```

Reply:
Okay. Thanks